containers-sig-benchmarks/src/Main.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Main where
+
+import qualified Map.Contrib.Bench.Int.RO as BIRO
+import qualified Map.Contrib.Bench.Ord.RO as BORO
+import qualified Map.Contrib.Bench.Hash.RO as BHRO
+import Data.Proxy (Proxy (..))
+
+-- Map key type for benchmarks. It can be Int, Char or any
+-- type that obeys to the constraint:
+-- (Enum k, Bounded k, Key k, NFData (Map k Int))
+type K = Int
+
+main :: IO ()
+main = do
+  BIRO.benchmark "intmap" (Proxy @K)
+
+  BORO.benchmark "ordmap" (Proxy @K)
```

Comment:
Looks like benchmarks for `primitive-containers` are missing here...

Reply:
Yep, I included this matter in "Questions" section of the PR description, I simply could not figure out how I could implement it since Prim uses `Data.Map.Lifted.Lifted` inside.